readme.txt

```diff
@@ -247,6 +247,14 @@ Yes, through the use of our commercial addon called [Recurring Payments](https:/
 11. Google Pay checkout
 
 == Changelog ==
+= 2.10.6, June 22, 2021 =
+* New: Added license upgrade notices to EDD admin pages.
+* Improvement: The discount API endpoint now includes `excluded_products`.
+* Improvement: The Hummingbird cache plugin is now detected in `edd_is_caching_plugin_active()`.
+* Fix: Currency symbol is prefixed with minus sign in PHP 8 on some interfaces.
```

Comment:
The other messages' wording is the desired/fixed behavior, but this is not. Should it say "was prefixed" instead?

```suggestion
* Fix: Currency symbol was prefixed with minus sign in PHP 8 on some interfaces.
```